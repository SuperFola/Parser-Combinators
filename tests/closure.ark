(fun (&a) 1)
(fun (&a
&b) 2)
(fun (a &b) 3)
(fun (a b &c &d) 4)