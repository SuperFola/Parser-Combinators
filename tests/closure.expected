( Keyword:fun ( Capture:a ) Number:1 )
( Keyword:fun ( Capture:a Capture:b ) Number:2 )
( Keyword:fun ( Symbol:a Capture:b ) Number:3 )
( Keyword:fun ( Symbol:a Symbol:b Capture:c Capture:d ) Number:4 )